crates/forge/tests/e2e/coverage.rs

```diff
@@ -36,7 +36,7 @@ fn test_coverage_project_and_pass_args() {
 }
 
 #[test]
-#[cfg_attr(feature = "scarb_2_8_3", ignore)]
+#[cfg_attr(not(feature = "scarb_2_7_1"), ignore)]
 fn test_fail_on_scarb_version_lt_2_8_0() {
```

Comment:
Let's maybe rename it then

Reply:
I believe this name is still valid as we want to test that it fails for versions less than `2.8.0`. We're currently using Scarb `2.7.1` for this, but we could technically use any other version.